standard_readme.md

```diff
@@ -0,0 +1,75 @@
+# Name of the project
+
+Description of the project. What is it solving? Who are the users?
+
+URL to the project
```

Comment:
I'd make this URL**S** given the staging/production setup we generally use.

Reply:
URLS to related github repositories for when there are frontend/backend repos.